src/app/pages/create-wallet/create-wallet.tsx

```diff
@@ -1,7 +1,158 @@
+                            </LayoutGrid.Cell>
+                            <LayoutGrid.Cell cols={4} className="secret-word">
+                                {state.words[8]}
+                            </LayoutGrid.Cell>
+                        </LayoutGrid.Inner>
+                        <LayoutGrid.Inner>
+                            <LayoutGrid.Cell cols={4} className="secret-word">
+                                {state.words[9]}
+                            </LayoutGrid.Cell>
+                            <LayoutGrid.Cell cols={4} className="secret-word">
+                                {state.words[10]}
+                            </LayoutGrid.Cell>
+                            <LayoutGrid.Cell cols={4} className="secret-word">
+                                {state.words[11]}
+                            </LayoutGrid.Cell>
+                        </LayoutGrid.Inner>
+                        <LayoutGrid.Inner>
+                            <LayoutGrid.Cell cols={12}>
+                                <Card.ActionButton
+                                    ripple
+                                    className="copy-to-clipboard"
+                                    onClick={() => this.copyToClipboard()}
+                                >
+                                    COPY TO CLIPBOARD
+                                </Card.ActionButton>
+                            </LayoutGrid.Cell>
+                        </LayoutGrid.Inner>
+                    </LayoutGrid>
+                </div>
+            );
+        } else {
+            wordsList = (
+                <div
+                    id="hidden-words"
+                    onClick={() => {
+                        this.revealWords();
+                    }}
+                >
+                    <img id="eye" src="/assets/eye.png" />
+                    <p id="hidden-words-text">Click Here to Reveal Secret Phrase</p>
+                </div>
+            );
+        }
+
+        return (
+            <div className="create-wallet-page">
+                <div class="padding-left-right">
+                    <LayoutGrid className="mdc-top-app-bar--fixed-adjust" />
+                    <LayoutGrid.Cell cols={12} className="center">
+                        <h1 class="title mdc-typography--headline2">Backup Secret Phrase</h1>
+                    </LayoutGrid.Cell>
+                    <LayoutGrid.Cell cols={12} className="center">
+                        <p className="left-text margin-top mdc-typography--caption">
+                            WARNING: Never disclose your secret phrase. Anyone with this phrase can
+                            take your funds forever.
+                        </p>
+                    </LayoutGrid.Cell>
+                    <LayoutGrid.Cell className="center">{wordsList}</LayoutGrid.Cell>
+                    <LayoutGrid.Cell cols={12} className="center">
+                        <p className="left-text mdc-typography--caption">
```

Comment:
I know that when you wrote this code we didn't have translation support, please use Translate component. If you think you need help i will help you with this.